src/tests/unit/tests/bug-filing/github/github-bug-filing-service.test.tsx

```diff
@@ -75,8 +75,6 @@ describe('GithubBugFilingServiceTest', () => {
     });
 
     describe('create bug filing url', () => {
-        it.each(invalidTestSettings)('with %o', settings => {
-            expect(GitHubBugFilingService.createBugFilingUrl(settings, null)).toBeNull();
-        });
+        expect(GitHubBugFilingService.createBugFilingUrl).not.toBeNull();
```

Comment:
This doesn't looks correct... I guess the test should assert the createBugFilingUrl on GitHubFilingService is the same that we expect it to be... just not being null doesn't seems enough.